pages/tidal tools/analyze_database.md

```diff
@@ -26,30 +26,20 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
  - Migration difficulty score is calculated based on a weighted model
 
 ## Getting Started
-- Be sure to have [logged in](tidal-tools.html#login) to your Tidal Migrations account via Tidal Tools.
+- Enable the Database Analysis feature for your account:
+  https://yoursubdomainhere.tidalmg.com/#/settings - See the Database Analysis section, under Preferences.
+- `tidal login`
+  Be sure to have [installed](tidal-tools.html) and [logged in](tidal-tools.html#login) to your Tidal Migrations account via Tidal Tools.
 - Install [Docker CE](https://docs.docker.com/v17.12/install/), it is compatible with most OSs, select the one you need. Version 17.12 or later will work with Tidal Tools. [Why Docker?](#why-docker)
 - You will also need a few authentication and configuration details for the database:
-    - engine - the database vendor, either `Oracle` or `SQL Server`
-    - host - the host that the database is running on and is accessible to access
-    - port - the port that the host has open and the database can accept connections on, the defaults are Oracle 1521, and SQL Server 1433
-    - db_name - the name of the database that will be analyzed, as it is defined within the database engine itself. ie. the value that is used by applications to connect to the database by name.
-    - user - a username to authenticate and access the database with
-    - pass - a password for the corresponding user
-    - name - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
-
-You can define these values in a YAML configuration file, `config.yml`:
-```
-databases:
-  - name: My-Test-DB
-    engine: Oracle
-    host: my-db-host.com
-    port: 1521
-    db_name: orcl
-    user: tidal
-    password: “yoursecurepassword1234!”
-```
-
-The analysis only requires access to metatables of the database and none of the application data itself.
+  - id - The id of the database from your Tidal Migrations account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal Migrations, the URL will show https://demo2.tidalmg.com/#/database_instances/111 in this case 111 is the database instance ID.
+  - engine - The database vendor, either `Oracle` or `SQL Server`, it is not case sensitive.
+  - host - The host that the database is running on and is accessible to access.
```

Comment:
“accessible to access” looks tautological to me

Reply:
Thanks, Petr.

WDYT about:
`The hostname of the server that the database is located on and is accessible via a network connection from your current device and location.`

Also removed the term itself from the definition.

Reply:
LGTM!